### PR TITLE
F/shims for ie8

### PR DESCRIPTION
**Pull request by alavers**

### CODE REVIEW

**Comment by alavers** [comment]
Note that this change only puts the javascript in working order. The HTML+CSS render quite badly, but it's functional.

Also, in IE8 the appUserId is persisted, but any existing conversation messages fail to get fetched on page refresh. To be fixed later.

File: src/js/main.js [hunk 1]
```diff
@@ -2,12 +2,14 @@
 
 'use strict';
 
+require('./compat');
```
Comment (juliangarritano) [inline src/js/main.js]: Maybe drop these in bootstrap instead?

File: example/template/demo.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8">
     <title>Conversation Demo</title>
     <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
-    <script src="/dist/supportkit.js"></script>
+    <script src="/dist/WIDGET_CODE"></script>
```
Comment (juliangarritano) [inline example/template/demo.html]: Cool.

**Comment by juliangarritano** [comment]
:+1:

File: src/js/main.js (hunk 1 above)
Comment (lemieux) [inline src/js/main.js]: +1

**Comment by lemieux** [comment]
:+1:

**Pull request merged**